js/index.js

```diff
@@ -233,7 +233,7 @@ var renderPage = function(data) {
     });
 }
 
-$.getJSON(window.location.origin+"/config.json", function(config) {
+$.getJSON("config.json", function(config) {
```

Comment:
That is related to misconfiguration of the server. Nothing wrong with the existing implementation